pkg/render/intrusion_detection.go

```diff
@@ -217,7 +217,7 @@ func (c *intrusionDetectionComponent) Objects() ([]client.Object, []client.Objec
 		objs = append(objs, c.adDetectorPodTemplates()...)
 	}
 
-	if !c.cfg.ManagedCluster {
+	if !c.cfg.ManagedCluster && !operatorv1.IsFIPSModeEnabled(c.cfg.Installation.FIPSMode) {
```

Comment:
I'm a little curious about this. Why can't IDS use / configure elasticsearch in none fips mode? I thought IDS was pretty reliant on ES, do we basically disable all IDS features in FIPS mode?